packages/core/src/configuration.ts

```diff
@@ -151,11 +154,22 @@ const axisChart: AxisChartOptions = Tools.merge({}, chart, {
 	timeScale,
 	grid,
 	zoomBar: {
+		zoomRatio: 0.4,
+		minZoomRatio: 0.01,
 		top: {
 			enabled: false,
 			type: ZoomBarTypes.GRAPH_VIEW
 		}
-	} as ZoomBarsOptions
+	} as ZoomBarsOptions,
+	toolbar: {
+		enabled: false,
+		overflowMenuItems: {
+			resetZoom: {
+				enabled: false,
+				text: "Reset zoom"
+			} as ToolbarOverflowMenuItemOptions
+		} as ToolbarOverflowMenuItems
+	} as ToolbarOptions
```

Comment:
So what's been discussed with Josh Kimmel and the rest of the dataviz guild would work a bit differently than specified here.

**The user would need a way of defining how many icons to show and how many to move into the overflow**.

This means that each item should have a name & icon, and the items can no longer be called `overflow` items but more so just `controls` in general.

That way the user can define a max number (e.g. 2) which means that the zoom in & zoom out buttons would show up as icons and everything else would go inside the overflow menu.

Additionally, the user needs to have a way of changing the order (e.g. define the order as `zoom in`, `zoom out`, `reset zoom`, `make fullscreen`, `print` and choose the limit as 3 which'll mean they'll only see zoom in, zoom out & reset zoom as icons)

Not sure if this is the best route to go but it's what design is asking for. I'll tag @jeanservaas here for feedback